lua/autorun/nl_library.lua

```diff
@@ -0,0 +1,460 @@
+// The tazer is a dependancy at this point //
+NLL = {}
+
+//NetworkStrings//
+if SERVER then
+	util.AddNetworkString("nllstartview")
+	util.AddNetworkString("nllendview")
```

Comment:
Since these two networkstrings have been removed, you can remove nllstartview and nllendview here